### PR TITLE
ZG lowMLL inclusive and PtG-130 samples for UL 

### PR DESCRIPTION
**Pull request by bobaekim**
ZG lowMLL inclusive and PtG-130 samples for Summer20 UL (2017, 2018)

Modified run cards based on [1] and [2]
- updated PDF ; lhaid, reweight_PDF
- requested ptG-10 instead of ptG-15 for the same as previous 2016 sample (inclusive)

dir for ZG lowMLL inclusive
- bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_pta15_5f_NLO_FXFX/

dir for ZG lowMLL PtG-130
- bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_pta130_5f_NLO_FXFX/

[1] https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat
[2] https://github.com/cms-sw/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/13TeV/ZATo2LA01j_mll4_ptl0_pta130_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_pta130_5f_NLO_FXFX_run_card.dat

### CODE REVIEW

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+#***********************************************************************
+  tag_1     = run_tag ! name of the run 
+#***********************************************************************
+# Number of LHE events (and their normalization) and the required      *
+# (relative) accuracy on the Xsec.                                     *
+# These values are ignored for fixed order runs                        *
+#***********************************************************************
+  1500 = nevents ! Number of unweighted events requested 
+ 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents)
+    20 = nevt_job! Max number of events per job in event generation.
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat:30]: Change this to -1

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+# F        = fixed_QES_scale  ! if .true. use fixed Ellis-Sexton scale
+# 91.188   = QES_ref_fixed    ! fixed Ellis-Sexton reference scale
+  -1 = dynamical_scale_choice ! Choose one (or more) of the predefined
+ 		! dynamical choices. Can be a list; scale choices beyond the
+		! first are included via reweighting		 
+ 1.0  = muR_over_ref     ! ratio of current muR over reference muR
+ 1.0  = muF_over_ref  ! ratio of current muF over reference muF
+# 1        = muF1_over_ref    ! ratio of current muF1 over reference muF1
+# 1        = muF2_over_ref    ! ratio of current muF2 over reference muF2
+# 1        = QES_over_ref     ! ratio of current QES over reference QES
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat:95]: Please remove the comment-outed lines with #

File: bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+  1.0 = rw_rscale ! muR factors to be included by reweighting
+  1.0 = rw_fscale ! muF factors to be included by reweighting
+ True   = reweight_scale   ! reweight to get scale dependence
+#  0.5     = rw_Rscale_down   ! lower bound for ren scale variations
+#  2.0     = rw_Rscale_up     ! upper bound for ren scale variations
+#  0.5     = rw_Fscale_down   ! lower bound for fact scale variations
+#  2.0     = rw_Fscale_up     ! upper bound for fact scale variations
+  $DEFAULT_PDF_MEMBERS  = reweight_PDF     ! reweight to get PDF uncertainty
+ 292201   = PDF_set_min      ! First of the error PDF sets 
+ 292302   = PDF_set_max      ! Last of the error PDF sets
```
Comment (sihyunjeon) [inline bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX/ZATo2LA01j_mll4_ptl0_5f_NLO_FXFX_run_card.dat:110]: 292201   = PDF_set_min      ! First of the error PDF sets 
 292302   = PDF_set_max      ! Last of the error PDF sets

This should be removed

**Comment by sihyunjeon** [comment]
Please make the same changes I mentioned above to the other run card as well

**Comment by bobaekim** [comment]
> Please make the same changes I mentioned above to the other run card as well

**Comment by sihyunjeon** [comment]
Sorry

   1500 = nevents ! Number of unweighted events requested 
 0.001 = req_acc ! Required accuracy (-1=auto determined from nevents) 

These are fine as they were,

     20 = nevt_job! Max number of events per job in event generation. 
I meant this should be changed to -1

**Comment by sihyunjeon** [comment]
@agrohsje They look okay to me

**Comment by bobaekim** [comment]
> @agrohsje They look okay to me

Thank you for your time. :)

**Comment by agrohsje** [comment]
Hi @bobaekim . Looks good to me. Two small questions. Is the one 15 < ptgamma > 130 and the other >130 ? Or are the samples independent? Suggest to indicate the photon cut in the name for 15 as you do for 130. Thanks!

**Comment by agrohsje** [comment]
ping @bobaekim . please comment. thanks!

**Comment by bobaekim** [comment]
> Hi @bobaekim . Looks good to me. Two small questions. Is the one 15 < ptgamma > 130 and the other >130 ? Or are the samples independent? Suggest to indicate the photon cut in the name for 15 as you do for 130. Thanks!

I'm sorry for late reply. I modified name as your comments. Thank you !

**Comment by sihyunjeon** [comment]
@bobaekim A bit more general question.
Is there reason to make gridpacks for inclusive and ptG>130?
Would it make more sens to make the two orthogonal (ptG<130 and ptG>130)

**Comment by bobaekim** [comment]
> @bobaekim A bit more general question.
> Is there reason to make gridpacks for inclusive and ptG>130?
> Would it make more sens to make the two orthogonal (ptG<130 and ptG>130)

I'm sorry for late reply.
I used ZG inclusive sample applying ptG < 130 GeV in gen-level. 
ptG>130 sample is for more statistics at high pt region.
So, I think that your comment is proper for my analysis.
In summary, I need two samples : 
- (inclusive) mll > 4 GeV && ptG > 15 GeV  -> mll > 4 GeV && 15 GeV < ptG <= 130 GeV
- (PtG-130) mll > 4 GeV && ptG > 130 GeV
Thank you.

**Comment by sihyunjeon** [comment]
Closing this request as we (EXO) agreed to make some collective efforts with SMP and #2800

**Pull request closed without merging**